packages/reports/tests/acceptance/report-test.js

```diff
@@ -1509,6 +1542,182 @@ module('Acceptance | Navi Report', function(hooks) {
       ['Day', 'Property'],
       'Removing a dimension as a filter and dimension changes the selected items'
     );
+
+    dimensionItem = await getItem('dimension', 'Operating System');
+
+    assert.ok(dimensionItem.item.querySelector('.fa-plus-circle'), 'Removed dimension row has a plus icon again');
```

Comment:
Since we didn't check that it wasn't a plus icon, maybe change the assert wording or assert that it was a minus icon?

Reply:
I didn't explicitly check it was a minus icon since that's how `getAllSelected` selects the elements but I'm fine with changing the wording